modules/core/src/main/scala/lucuma/itc/client/ItcSpectroscopyInput.scala

```diff
@@ -185,24 +185,26 @@ object ItcSpectroscopyInput {
         )
     }
 
-  implicit val EncoderConstraintSetModel: Encoder[ConstraintSetModel] =
-    (a: ConstraintSetModel) =>
+  implicit val EncoderConstraintSetModel: Encoder[ConstraintSet] =
+    (a: ConstraintSet) =>
       Json.obj(
         "imageQuality"    -> screaming(a.imageQuality),
         "cloudExtinction" -> screaming(a.cloudExtinction),
         "skyBackground"   -> screaming(a.skyBackground),
         "waterVapor"      -> screaming(a.waterVapor),
         "elevationRange"  -> (a.elevationRange match {
-          case AirmassRange(min, max)             =>
+          case ElevationRange.AirMass(min, max)             =>
             Json.obj(
+              // TODO: we should switch this to airMass to be consistent
```

Comment:
@cquiroz just calling this to your attention.  I switched to `airMass` to be consistent with `lucuma-core` so we might want to switch here too?